.env.example

```diff
@@ -98,3 +98,5 @@ AWS_SECRET=
 AWS_REGION=us-east-1
 AWS_BUCKET=
 AWS_SERVER=
+
+2FA_ENABLED=false
```

Comment:
Can you add a comment here on what 2FA is and does?

Reply:
OK thanks